contents/docs/migrate/migrate-to-another-self-hosted-instance.mdx

```diff
@@ -4,30 +4,26 @@ sidebar: Docs
 showTitle: true
 ---
 
-import ExportingEventsSnippet from './snippets/exporting-events'
-
-> If you're attempting this migration, feel free to ask questions and provide feedback via the [PostHog Communty Slack workspace](/slack) or [a GitHub issue](https://github.com/PostHog/posthog.com/issues). You should also be aware that **some of the steps on this document are potentially destructive**! Proceed with caution.
-
-**PostHog backed by PostgreSQL is now deprecated**. We will continue to provide support to Postgres backed installs but we strongly encourage you to migrate to PostHog backed by ClickHouse for [vastly superior performance](/docs/self-host/postgres-vs-clickhouse), as well as to receive new features and continued support in the future.
+import MigratingEventsSnippet from './snippets/migrating-events'
 
 > Note: If you're looking for a more hands-off experience we recommend migrating to [PostHog Cloud](/docs/migrate/migrate-between-cloud-and-self-hosted).
 
-## Requirements
+## Migrating event data using the Replicator App
 
--   You should have a **clean** [ClickHouse-backed PostHog instance up and running](/docs/self-host). Your new PostHog instance should have **no ingested events**. We recommend using a fresh and unused installation.
--   Your old and new instances should both be running the **exact** same version of PostHog (minimum `1.30.0`).
+<MigratingEventsSnippet />
 
-> **Having difficulties upgrading your Postgres-backed instance due to long-running migrations?** Check out [this guide](/docs/self-host/postgres-upgrade-migrations).
+## Migrating non-event data
 
-## Approach
+To migrate the non-event data (such as user information, feature flags, dashboard configurations, etc) we'll need to create a data dump and load that into the new instance. This non-event data is stored in Postgres.
 
-ClickHouse-backed PostHog instances uses a Postgres database to store data that is not used in analytical queries, such as user information, feature flags, dashboard configurations, etc. However, tables that contain data used to run analytical queries such as events, persons, and person distinct IDs is now stored in ClickHouse.
+This is only officially supported for importing events into a new cloud instance of PostHog or an enterprise self-hosted instance using Kubernetes. If you're using a different deployment method, you can still try this approach but we can't guarantee it will work.
```

Comment:
```suggestion
> This is only officially supported for importing events into a new cloud instance of PostHog or an enterprise self-hosted instance using Kubernetes. If you're using a different deployment method, you can still try this approach but we can't guarantee it will work.
```

feels worth highlight to me